OlinkAnalyze/R/olink_normalization_n.R

```diff
@@ -0,0 +1,892 @@
+#' Bridge and/or subset normalization of all proteins among multiple NPX
+#' projects.
+#'
+#' This function normalizes pairs of NPX projects (data frames) using shared
+#' samples or subsets of samples.\cr\cr
+#'
+#' This function is a wrapper of olink_normalization_bridge and
+#' olink_normalization_subset.\cr\cr
+#'
+#' The input of this function is a tibble that contains all the necessary
+#' information to normalize multiple NPX projects. This tibble is called the
+#' normalization schema. The basic idea is that every row of the data frame is
+#' a separate project to be normalized. We assume that there is always one
+#' baseline project that does not normalize to any other. All other project
+#' normalize to one or more projects. The function handles projects that are
+#' normalized in a chain, for example:
+#' \itemize{
+#'    \item{1.} project 2 normalizes to project 1, and project 3 normalizes to
+#'    project 2.
+#'    \item{2.} project 2 normalizes to project 1, and project 3 normalizes to
+#'    the combined data frame of projects 1 and 2 (that is already normalized).
+#' }
+#'
+#' The function can also handle a mixed schema of bridge and subset
+#' normalizations.
+#'
+#' Specifications of the normalization schema data frame:
+#' \itemize{
+#'    \item{order:} should strictly be a numeric or integer array with unique
```

Comment:
Looking at the examples (Havent reviewd code yet, dont this always have to be an integer? Remove the "a numeric or" part?